tests/wpunit/Tribe/Events/Models/Event_ModelTest.php

```diff
@@ -46,6 +48,28 @@ public function it_should_return_past() {
 		$this->assertTrue( $event->is_past );
 	}
 
+	/**
+	 * @test
+	 */
+	public function it_should_return_is_now() {
+		$mock  = $this->get_mock_event( 'events/single/1.json' );
```

Comment:
No need to use a mock event for anything that is not a snapshot: use a real Event created using `tribe_events()->set_args( ... )->create()`.